<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoDataBackgroundEditDialog</class>
 <widget class="QDialog" name="GeoDataBackgroundEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="minRadioButton">
     <property name="text">
      <string>Minimum value of geographic data</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="maxRadioButton">
     <property name="text">
      <string>Maximum value of geographic data</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="customRadioButton">
       <property name="text">
        <string>Custom value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="GridAttributeEditWidgetContainer" name="customWidget" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GridAttributeEditWidgetContainer</class>
   <extends>QWidget</extends>
   <header>pre/gridcond/base/gridattributeeditwidgetcontainer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GeoDataBackgroundEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GeoDataBackgroundEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
